extensions/aria_extension_tosca/simple_v1_0/modeling/__init__.py

```diff
@@ -85,9 +85,14 @@ def create_service_template_model(context): # pylint: disable=too-many-locals,to
     policies = context.presentation.get('service_template', 'topology_template', 'policies')
     if policies:
         for policy in policies.itervalues():
-            if model.policy_types.get_descendant(policy.type).role == 'plugin':
+            role = model.policy_types.get_descendant(policy.type).role
```

Comment:
what about policies which are neither plugin or workflows?

I think the expected behavior should be that any such policy will be available under `service.policies`, while the plugin and workflow ones won't be available under `service.policies`, no?